modules/workstation-cluster/variables.tf

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "annotations" {
+  description = "Workstation cluster annotations."
+  type        = map(string)
+  default     = {}
+}
+
+variable "display_name" {
+  description = "Display name."
+  type        = string
+  default     = null
+}
+
+variable "domain" {
+  description = "Domain."
+  type        = string
+  default     = null
+}
+
+variable "labels" {
+  description = "Workstation cluster labels."
+  type        = map(string)
+  default     = {}
+}
+
+variable "location" {
+  description = "Location."
+  type        = string
+  default     = null
+}
+
+variable "network" {
+  description = "Network ID."
+  type        = string
+}
+
+variable "private_cluster_config" {
+  description = "Private cluster config."
+  type = object({
+    enable_private_endpoint = optional(bool, false)
+    allowed_projects        = optional(list(string))
+  })
+  nullable = false
+  default  = {}
+}
+
+variable "project_id" {
+  description = "Cluster ID."
+  type        = string
+}
+
+variable "subnetwork" {
+  description = "Subnet ID."
+  type        = string
+}
```

Comment:
what about merging `var.network` and `var.subnetwork` in a single variable (e.g. `network_config`)